tests/test_integration.py

```diff
@@ -74,12 +69,20 @@ class Meta:
 
 class TestSwagger(unittest.TestCase):
 
-    def test_swagger_valid(self):
+    def test_swagger(self):
         try:
             utils.validate_swagger(spec)
         except exceptions.SwaggerError as error:
             self.fail(str(error))
 
+    def test_swagger_with_cache(self):
+        # Test if Swagger API documentation loads OK
+        # when cache all requests is enabled
+        os.environ['CACHE_ALL_REQUESTS'] = 'True'
```

Comment:
This line still needs to be removed from here as we've identified it's not appropriate in this test; we need it in a different test instead.